### PR TITLE
feat(env): read variables from .env file

### PR DESCRIPTION
**Pull request by gauravsaini**
read vars from env file 
otherwise use the fallback

### CODE REVIEW

File: src/utils/chain.ts [hunk 1]
```diff
@@ -6,9 +6,9 @@ export const createModel = (customApiKey: string) =>
   new OpenAI({
     openAIApiKey:
       customApiKey === "" ? process.env.OPENAI_API_KEY : customApiKey,
-    temperature: 0.9,
-    modelName: "gpt-3.5-turbo",
-    maxTokens: 300,
+      temperature: process.env.TEMPRATURE  || 0.9,
```
Comment (awtkns) [inline src/utils/chain.ts:9]: @gauravsaini thoughts on prefixing this with OPENAI_

Comment (gauravsaini) [reply src/utils/chain.ts:9]: These `in future` can be with anything (not necessarily OPENAI )
I think AGENTGPT_ would be a better prefix 
Thoughts ?

Comment (awtkns) [reply src/utils/chain.ts:9]: @gauravsaini my thoughts with the prefix are that in the future we will most likely have multiple model that will be running at once in which case we would need to distinguish between tempratures

**Review by awtkns — CHANGES_REQUESTED** [review]

**Comment by mrayonnaise** [comment]
I have been working on something similar to this too. Maybe we can combine efforts? I am adding the MAX_LOOPS as a config item for phase 1 and then adding it as a setting in the UI in phase 2. My work in progress (WIP) is here:
https://github.com/reworkd/AgentGPT/pull/51

**Comment by gauravsaini** [comment]
@awtkns  : Done

File: src/utils/chain.ts [hunk 2]
```diff
@@ -6,9 +6,9 @@ export const createModel = (customApiKey: string) =>
   new OpenAI({
     openAIApiKey:
       customApiKey === "" ? process.env.OPENAI_API_KEY : customApiKey,
-    temperature: 0.9,
-    modelName: "gpt-3.5-turbo",
-    maxTokens: 300,
+      temperature: process.env.OPENAI_TEMPRATURE  || 0.9,
```
Comment (chozzz) [inline src/utils/chain.ts:9]: Is it a typo? `OPENAI_TEMPRATURE`?

**Pull request closed without merging**